cargo-spatial/src/local.rs

```diff
@@ -0,0 +1,53 @@
+use crate::config::Config;
+use crate::opt::*;
+use std::ffi::OsString;
+use std::path::*;
+use std::process;
+use tap::*;
+
+/// Prepares and launches a local deployment.
+///
+/// Before launching the deployment, this will first run code generation and build
+/// workers in the project. Assumes that the current working directory is the root
+/// directory of the project, i.e. the directory that has the `Spatial.toml` file.
+pub fn launch(_opt: &Opt, _local: &Local, launch: &LocalLaunch) {
+    let config = Config::load().expect("Failed to load configuration");
+
+    // Run codegen and such.
+    crate::codegen::run_codegen(&config).expect("Failed to run codegen");
+
+    // Use `cargo install` to build workers and copy the exectuables to the build
+    // directory.
+    //
+    // TODO: Manually copy the built executables instead of using `cargo install`.
+    // `cargo install` doesn't use the same build cache as normal builds, so it will
+    // sometimes result in unnecessary recompilation, which can slow down launch times.
+    if !launch.no_build {
+        let build_dir = PathBuf::from(&config.build_dir).join("debug");
```

Comment:
Should probably expose the ability to use `release` builds for local launch as well (although not default)